packages/block-editor/src/components/rich-text/index.js

```diff
@@ -269,7 +269,7 @@ export function RichTextWrapper(
 		readOnly || disableBoundBlock || shouldDisableForPattern;
 
 	const { getSelectionStart, getSelectionEnd, getBlockRootClientId } =
-		useSelect( blockEditorStore );
+		useSelect( blockEditorStore, [] );
```

Comment:
It seems like useSelect already accounts for this and the empty dependency array is not necessary.